messages/describe/edge_applications/errors.go

```diff
@@ -0,0 +1,8 @@
+package edge_applications
+
+import "errors"
+
+var (
+	ErrorGetApplication       = errors.New("Failed to get the Edge Application: %s. Check your settings and try again. If the error persists, contact Azion support")
+	ErrorConvertIdApplication = errors.New("The application ID you provided is invalid. The value must be an integer. You may run the 'azion list edge-application' ommand to check your application ID")
```

Comment:
Just a small typo. The `c` letter was missing. 
```suggestion
	ErrorConvertIdApplication = errors.New("The application ID you provided is invalid. The value must be an integer. You may run the 'azion list edge-application' command to check your application ID")
```